### DISCUSSION TITLE
lutris: cannot access whitelisted directories

### DISCUSSION THREAD

**Discussion by swrast** [netblue30/firejail#5855] [Q&A] [answered]
### Description
Whitelisting `${HOME}/documents/games` doesn't work (lutris shows error: permission denied), but whitelisting `${HOME}/Games` does.
It seems more like an lutris's profile issue because I can access `${HOME}/documents/games` from zsh via `firejail --whitelist='${HOME}/documents/games' zsh`.

### Steps to Reproduce

1. Set up `${HOME}/documents/games` whitelisting
1. Run  `firejail lutris`
2. Click on any button that invokes file chooser dialog window.
3. Select `~/documents` folder
4. See error `permission denied`

### Additional context

`~/.config/firejail/lutris.local`:
```
noblacklist ${HOME}/documents/games
whitelist ${HOME}/documents/games
```

![изображение](https://github.com/netblue30/firejail/assets/115649270/f91c64b4-9909-494c-9ace-8d15b0e5a6c3)

### Log

<details>
<summary>Output of <code>LC_ALL=C firejail firejail</code></summary>
<p>

Reading profile /etc/firejail/lutris.profile
Reading profile /home/asd/.config/firejail/lutris.local
Reading profile /etc/firejail/allow-python2.inc
Reading profile /etc/firejail/allow-python3.inc
Reading profile /etc/firejail/disable-common.inc
Reading profile /etc/firejail/disable-devel.inc
Reading profile /etc/firejail/disable-exec.inc
Reading profile /etc/firejail/disable-interpreters.inc
Reading profile /etc/firejail/disable-programs.inc
Reading profile /etc/firejail/disable-xdg.inc
Reading profile /etc/firejail/whitelist-common.inc
Reading profile /etc/firejail/whitelist-usr-share-common.inc
Reading profile /etc/firejail/whitelist-runuser-common.inc
Reading profile /etc/firejail/whitelist-var-common.inc
Seccomp list in: !modify_ldt, check list: @default-keep, prelist: unknown,
Parent pid 6922, child pid 6926
Warning: /var/lock not mounted
Warning: An abstract unix socket for session D-BUS might still be available. Use --net or remove unix from --protocol set.
Seccomp list in: !modify_ldt, check list: @default-keep, prelist: unknown,
Seccomp list in: !modify_ldt, check list: @default-keep, prelist: unknown,
Child process initialized in 71.76 ms
Warning: seccomp disabled, it requires a Linux kernel version 3.5 or newer.
Warning: an existing sandbox was detected. /usr/bin/lutris will run without any additional sandboxing features

(lutris:13): dbind-WARNING **: 21:09:11.882: AT-SPI: Error retrieving accessibility bus address: org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown
2023-06-12 21:09:12,211: Starting Lutris 0.5.13
2023-06-12 21:09:12,213: Running Intel Mesa driver 23.1.2 on Mesa Intel(R) Graphics (ADL GT2) (0x4628)
2023-06-12 21:09:12,213: GPU: 8086:4628 0000:0000 (i915 drivers)
2023-06-12 21:09:12,228: vulkaninfo not available, unable to list GPUs
2023-06-12 21:09:12,229: vulkaninfo not available, unable to list GPUs
2023-06-12 21:09:12,344: Unable to get locale
Traceback (most recent call last):
  File "/usr/lib/python3.11/site-packages/lutris/style_manager.py", line 77, in _call_cb
    values = obj.call_finish(result)
             ^^^^^^^^^^^^^^^^^^^^^^^
gi.repository.GLib.GError: g-dbus-error-quark: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown (2)

(lutris:13): dconf-WARNING **: 21:09:17.317: failed to commit changes to dconf: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown

(lutris:13): dconf-WARNING **: 21:09:17.335: failed to commit changes to dconf: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown

(lutris:13): dconf-WARNING **: 21:09:17.352: failed to commit changes to dconf: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown

(lutris:13): dconf-WARNING **: 21:09:17.370: failed to commit changes to dconf: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown

(lutris:13): dconf-WARNING **: 21:09:17.386: failed to commit changes to dconf: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown

(lutris:13): dconf-WARNING **: 21:09:17.403: failed to commit changes to dconf: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown

(lutris:13): dconf-WARNING **: 21:09:18.520: failed to commit changes to dconf: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown

(lutris:13): dconf-WARNING **: 21:09:18.539: failed to commit changes to dconf: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown

(lutris:13): dconf-WARNING **: 21:09:18.556: failed to commit changes to dconf: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown

(lutris:13): dconf-WARNING **: 21:09:18.573: failed to commit changes to dconf: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown

(lutris:13): dconf-WARNING **: 21:09:18.590: failed to commit changes to dconf: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown

(lutris:13): dconf-WARNING **: 21:09:18.608: failed to commit changes to dconf: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown

(lutris:13): dconf-WARNING **: 21:09:19.874: failed to commit changes to dconf: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown

(lutris:13): dconf-WARNING **: 21:09:19.891: failed to commit changes to dconf: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown

(lutris:13): dconf-WARNING **: 21:09:19.910: failed to commit changes to dconf: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown

(lutris:13): dconf-WARNING **: 21:09:19.928: failed to commit changes to dconf: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown

(lutris:13): dconf-WARNING **: 21:09:19.945: failed to commit changes to dconf: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown

(lutris:13): dconf-WARNING **: 21:09:19.962: failed to commit changes to dconf: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown

(lutris:13): dconf-WARNING **: 21:09:20.364: failed to commit changes to dconf: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown

(lutris:13): dconf-WARNING **: 21:09:36.007: failed to commit changes to dconf: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown

(lutris:13): dconf-WARNING **: 21:09:36.008: failed to commit changes to dconf: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown
2023-06-12 21:09:39,615: Shutting down Lutris

Parent is shutting down, bye...
</p>
</details>

**Comment by rusty-snake** [comment, marked as answer]
Assuming `${HOME}/documents` is your XDG_DOXUMENTS_DIR.

You need to `noblacklist ${HOME}/documents` (**`noblacklist ${DOCUMENTS}`**) rather than `${HOME}/documents/games`.


See 2. at https://github.com/netblue30/firejail/wiki/Creating-Profiles#common-mistakes
